### ISSUE TITLE
Add 'ssl_version' to DEFAULT_SSL_OPTIONS

### ISSUE THREAD

**Issue by todomodo** [iande/onstomp#23]
As a ruby developer
I should be able to specify :ssl_version when calling OnStomp.connect
And have it taking effect when creating SSL context

Example:
{code}
require 'onstomp'
OnStomp.connect('stomp+ssl://user:pass@myhost.com:61614', {:ssl=>{:ssl_version=>'SSLv3'}})
{code}

Above call will currently have no effect since :ssl_version is not part of DEFAULT_SSL_OPTIONS and will be ignored when setting SSL Context inside "def self.create_socket_ssl client":

{code}
DEFAULT_SSL_OPTIONS.keys.each do |k|
      context.**send**(:"#{k}=", ssl_opts[k]) if ssl_opts.key?(k)
    end
{code}

In other words custom settings are only applied if they correspond to a default value. Since there's no default for :ssl_version, it will get ignored
